src/formats.jl

```diff
@@ -61,7 +61,7 @@ function pretty_stats(io::IO, df::DataFrame;
     if name ∈ keys(col_formatters)
       push!(pt_formatters, ft_printf(col_formatters[name], col))
     else
-      typ = eltype(df[!, name])
+      typ = Missings.nonmissingtype(eltype(df[!, name]))
```

Comment:
What does this do if some values are missing?

Reply:
Here if `typ` is a `Union{Missing,Other}`, it returns `Other`. Otherwise, if `typ` is just `Other`, it returns `Other`.
This is necessary to correctly assess the supertype below.

Reply:
Got it. Thanks for explaining.